lib/demos/reference/transformations_demo.dart

```diff
@@ -3,10 +3,11 @@
 // found in the LICENSE file.
 
 import 'package:flutter/material.dart';
+import 'package:flutter/widgets.dart';
 import 'package:gallery/l10n/gallery_localizations.dart';
+
```

Comment:
Since you updated the demo, you'll need to update code segments so that the code viewer shows the correct code, https://github.com/flutter/gallery/blob/master/tool/codeviewer_cli/README.md

The CI will fail on this but its currently failing early because of #222

Reply:
Thanks, I was just going to ping you for help on getting CI green.  I updated the PR with master and updated the code segments, so hopefully it works now.